controllers/storagecluster/storagecluster_controller.go

```diff
@@ -144,14 +149,76 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	)
 
-	builder := ctrl.NewControllerManagedBy(mgr).
+	noobaaIgnoreTimeUpdatePredicate := predicate.Funcs{
+		UpdateFunc: func(e event.UpdateEvent) bool {
+			if e.ObjectOld == nil || e.ObjectNew == nil {
+				return false
+			}
+			oldObj := e.ObjectOld.(*nbv1.NooBaa)
+			newObj := e.ObjectNew.(*nbv1.NooBaa)
+
+			ignorePaths := func(path cmp.Path) bool {
+				switch path.String() {
+				case "ObjectMeta.ManagedFields", "ObjectMeta.ResourceVersion",
+					"Status.Conditions.LastHeartbeatTime", "Status.Conditions.LastTransitionTime":
```

Comment:
Can we please format this with one case per line to help with readability
```suggestion
				case "ObjectMeta.ManagedFields", 
					"ObjectMeta.ResourceVersion",
					"Status.Conditions.LastHeartbeatTime", 
					"Status.Conditions.LastTransitionTime":
```

Reply:
Done